src/main/docker/Dockerfile.jvm.staged

```diff
@@ -28,43 +28,25 @@ RUN mvn dependency:go-offline
 RUN grep artifactId /build/target/maven-archiver/pom.properties | cut -d '=' -f2 >.env-id
 
-# mv the uber jar in case the quarkus property quarkus.package.type=uber-jar is set
-RUN mv /build/target/$(cat .env-id)-$(cat .env-version)*.jar /build/target/quarkus-app-tmp/
-
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7-1085 
-
-ARG JAVA_PACKAGE=java-17-openjdk-devel
-ARG RUN_JAVA_VERSION=1.3.8
-ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
-# Install java and the run-java script
-# Also set up permissions for user `1001`
-RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} \
-    && microdnf update \
-    && microdnf clean all \
-    && mkdir /deployments \
-    && chown 1001 /deployments \
-    && chmod "g+rwX" /deployments \
-    && chown 1001:root /deployments \
-    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
-    && chown 1001 /deployments/run-java.sh \
-    && chmod 555 /deployments/run-java.sh \
-    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
+# if this is an uber jar create a structure that looks the same as fast-jar with empty directories
+# this allows for the same dockerfile to be used with both
+RUN if [ ! -d /build/target/quarkus-app ] ; then mkdir -p /build/target/quarkus-app/lib; \
+     mkdir -p /build/target/quarkus-app/app; \
+     mkdir -p /build/target/quarkus-app/quarkus; \
+     mv /build/target/$(cat .env-id)-$(cat .env-version)*.jar /build/target/quarkus-app/ ; \
+     fi
 
+FROM registry.access.redhat.com/ubi8/openjdk-17-runtime:1.15
 # Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Dquarkus.http.port=8081 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 # We make four distinct layers so if there are application changes the library layers can be re-used
-COPY --from=0 --chown=1001 /build/target/quarkus-app-tmp/*.jar /deployments/export-run-artifact.jar
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*lib/ /deployments/lib/
-# Overwrite the uber jar if package type is normal
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*.jar /deployments/export-run-artifact.jar 
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*app/ /deployments/app/
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*quarkus/ /deployments/quarkus/
-
+COPY --from=0 --chown=1001 /build/target/quarkus-app/lib/ /deployments/lib/
+COPY --from=0 --chown=1001 /build/target/quarkus-app/*.jar /deployments/export-run-artifact.jar
+COPY --from=0 --chown=1001 /build/target/quarkus-app/app/ /deployments/app/
+COPY --from=0 --chown=1001 /build/target/quarkus-app/quarkus/ /deployments/quarkus/
 EXPOSE 8081
-USER 1001
+ENTRYPOINT ["/opt/jboss/container/java/run/run-java.sh", "--server.port=8081"]
```

Comment:
@stuartwdouglas this `"--server.port=8081` doesnt seem to work, just tried on RHTAP/HAC and locally on docker.

Reply:
It seems the app is starting on port 8080 instead of 8081

RHTAP:
```

INFO exec  java -XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+ExitOnOutOfMemoryError -cp "." -jar /deployments/export-run-artifact.jar --server.port=8081
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2023-04-19 19:20:18,215 WARN  [io.qua.net.run.NettyRecorder] (Thread-0) Netty DefaultChannelId initialization (with io.netty.machineId system property set to c7:0d:a4:6b:27:f0:f4:35) took more than a second
2023-04-19 19:20:24,214 INFO  [io.quarkus] (main) code-with-quarkus 1.0.0-SNAPSHOT on JVM (powered by Quarkus 1.13.2.Final) started in 16.700s. Listening on: http://0.0.0.0:8080
2023-04-19 19:20:24,214 INFO  [io.quarkus] (main) Profile prod activated. 
2023-04-19 19:20:24,214 INFO  [io.quarkus] (main) Installed features: [cdi, kubernetes, micrometer, resteasy, smallrye-health]
```

Locally on docker:
```
$ docker run -i --rm -p 8081:8081 quarkus/code-with-quarkus-jvm
INFO exec  java -XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+ExitOnOutOfMemoryError -cp "." -jar /deployments/export-run-artifact.jar --server.port=8081
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2023-04-19 19:29:16,229 INFO  [io.quarkus] (main) code-with-quarkus 1.0.0-SNAPSHOT on JVM (powered by Quarkus 1.13.2.Final) started in 0.759s. Listening on: http://0.0.0.0:8080
2023-04-19 19:29:16,267 INFO  [io.quarkus] (main) Profile prod activated. 
2023-04-19 19:29:16,268 INFO  [io.quarkus] (main) Installed features: [cdi, kubernetes, micrometer, resteasy, smallrye-health]
^C2023-04-19 19:30:41,706 INFO  [io.quarkus] (Shutdown thread) code-with-quarkus stopped in 0.036s
```

Reply:
That was a copy/paste error, but the real issue is that I did not update JAVA_OPTIONS to JAVA_OPTS